Tests/Integration/AuditSettingRule.Integration.tests.ps1

```diff
@@ -8,21 +8,21 @@ try
             @{
                 query        = "SELECT * FROM Win32_LogicalDisk WHERE DriveType = '3'"
                 property     = 'FileSystem'
-                value        = 'NTFS|ReFS'
+                desiredvalue = 'NTFS|ReFS'
                 operator     = '-match'
                 checkContent = 'Open the Computer Management Console.
-    Expand the "Storage" object in the Tree window.
-    Select the "Disk Management" object.
+                    Expand the "Storage" object in the Tree window.
```

Comment:
I can't seem to comment on code in this file not modified in this PR, but since we are tabbing the checkcontent here lines, the checkcontent starting on lines 29 and 71 should be tabbed as well for consistency.

Reply:
Done